packages/protoplugin-test/src/plugin-edition-support.test.ts

```diff
@@ -70,7 +70,7 @@ describe("editions support in plugins", () => {
       generateAny() {},
     });
     expect(res.minimumEdition).toBe(minimumEdition);
-    expect(res.minimumEdition).toBe(minimumEdition);
+    expect(res.maximumEdition).toBe(maximumEdition);
```

Comment:
This test is supposed to verify min and max edition, but it was testing min twice. This fixes the test.